bricks/_common/modules/_builtin_port_view.py

```diff
@@ -124,8 +124,9 @@ def update_dc_motor(port, type_id):
 
 
 # Any unknown Powered Up device.
-def unknown_pup_device(port, type_id, dev):
+def unknown_pup_device(port, type_id):
     while True:
+        dev = PUPDevice(port)  # reset on the un-plug of the unknown device
```

Comment:
```suggestion
        PUPDevice(port)  # reset on the un-plug of the unknown device
```

`dev` isn't used, so can be left out.